cql3/statements/describe_statement.cc

```diff
@@ -244,11 +252,27 @@ description index(const data_dictionary::database& db, const sstring& ks, const
     return description(db.real_database(), **idx, with_internals);
 }
 
+std::optional<description> describe_cdc_log_table(const data_dictionary::database& db, const sstring& ks, const sstring& base_name) {
+    auto table = db.try_find_table(ks, cdc::log_name(base_name));
+    if (!table) {
+        dlogger.warn("Couldn't find cdc log table for base table {}.{}", ks, base_name);
+        return std::nullopt;
+    }
+
+    auto schema = table->schema();
+    auto alter_statement = schema->alter_table_with_options(db.real_database());
+    return description(db.real_database(), *schema, std::move(alter_statement));
+}
+
 future<std::vector<description>> table(const data_dictionary::database& db, const sstring& ks, const sstring& name, bool with_internals) {
     auto table = db.try_find_table(ks, name);
     if (!table) {
         throw exceptions::invalid_request_exception(format("Table '{}' not found in keyspace '{}'", name, ks));
     }
+    if (cdc::is_log_for_some_table(db.real_database(), ks, name)) {
+        // we want to hide cdc log table from the user
+        throw exceptions::invalid_request_exception(format("CDC log table {}.{} cannot be described.", ks, name));
```

Comment:
I think the PR description is outdated. You wrote:

> If someone knows how the prefix looks like, those tables can be described directly with `DESC TABLE t_scylla_cdc_log`.

However, this code here suggests otherwise. Please update the PR description.

Reply:
> "CDC log table {}.{} cannot be described."

This is not helpful enough for the user. A user wants to describe a CDC table - sounds like a reasonable request - but Scylla tells him that he just cannot. Now what? The user will just be frustrated.

You should explain the reason why this is not allowed. Even better - anticipate the most common reason(s) why the user would like to do that and provide an alternative for their problem.

Reply:
> "CDC log table {}.{} cannot be described."

Moreover, I think this change is important enough to be noted in the commit message, and have the reasoning behind it explained in the message.

Reply:
> I think the PR description is outdated. You wrote:
> 
> > If someone knows how the prefix looks like, those tables can be described directly with `DESC TABLE t_scylla_cdc_log`.
> 
> However, this code here suggests otherwise. Please update the PR description.

This issue is not resolved yet. I can still see that an exception is thrown when one tries to describe a cdc log table directly, your test also verifies that the cdc log table __cannot__ be described. It will be confusing to see the PR or a commit with a description which suggests something opposite to what was implemented.